### PR TITLE
Implement and use wait_for_good_pings for autoix tests

### PR DESCRIPTION
**Pull request by codablock**
This should hopefully fix the flaky tests on Travis. Let's see what the next build says...

### CODE REVIEW

File: qa/rpc-tests/dip3-deterministicmns.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ def run_test(self):
             self.nodes[0].generate(1)
         self.test_fail_create_protx(self.nodes[0])
 
+        self.wait_for_good_pings()
```
Comment (UdjinM6) [inline qa/rpc-tests/dip3-deterministicmns.py]: no nodes?

Comment (codablock) [reply qa/rpc-tests/dip3-deterministicmns.py]: Yeah...I shouldn't push 2 minutes before leaving my home...Fixed.

**Comment by UdjinM6** [comment]
Btw, why `Move wait_for_good_pings to BitcoinTestFramework`? Maybe inherit `DIP3Test` class from `DashTestFramework` instead? (and dip3 test could also be refactored to reuse some of `DashTestFramework` functions too later)

**Comment by codablock** [comment]
@UdjinM6 yeah it is too general and might end up being useful in normal tests as well

**Comment by codablock** [comment]
Tests have been fixed through other PRs, closing this.

**Pull request closed without merging**